### ISSUE TITLE
Issues   using IE 11 with NVDA  master-11139.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5003]
**Reported by Brendon22 on 2015-03-26 20:18**
When the IE Window is at it's size, the NVDA links list will not come up and tracking in edit  fields does not work? But when it is Maximised this all works again?

**Comment by Adriani90** [comment]
@Brendon22 are you still having this issue? I cannot reproduce it with NVDA 2018.4.1 in IE11.

**Comment by Adriani90** [comment]
We don't have updates on this issue since years and I cannot reproduce. Closing as works for me and abandoned. Plese comment on this if you are still having this issue and we can reopen.
